tools/archetype/binding/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -11,21 +10,48 @@
 		<requiredProperty key="namespace">
 			<defaultValue>org.eclipse.smarthome</defaultValue>
 		</requiredProperty>
-        <requiredProperty key="author">
+		<requiredProperty key="author">
 		</requiredProperty>
-        <requiredProperty key="bindingId">
+		<requiredProperty key="bindingId">
+		</requiredProperty>
+		<requiredProperty key="bindingIdCamelCase">
 		</requiredProperty>
-        <requiredProperty key="bindingIdCamelCase">
-		</requiredProperty>        
 	</requiredProperties>
 
 	<fileSets>
 		<fileSet filtered="true" packaged="false" encoding="UTF-8">
 			<directory></directory>
 			<includes>
-				<include>**/*.*</include>
+				<include>cfg/*.*</include>
```

Comment:
is cfg/*.* filtered twice for any reasons (see below)? Otherwise I'd remove his block here.